app/angular/src/client/preview/render.ts

```diff
@@ -1,32 +1,41 @@
 import { StoryFn } from '@storybook/addons';
-import { RendererService } from './angular-beta/RendererService';
 
 import { renderNgApp } from './angular/helpers';
 import { StoryFnAngularReturnType } from './types';
 import { Parameters } from './types-6-0';
 
-// add proper types
-export default function render({
+import { RendererFactory } from './angular-beta/RendererFactory';
+
+const rootElement = global.document.getElementById('root');
+
+export const rendererFactory = new RendererFactory();
+
+export default async function renderMain({
   storyFn,
   showMain,
   forceRender,
   parameters,
+  targetDOMNode = rootElement,
+  id,
 }: {
   storyFn: StoryFn<StoryFnAngularReturnType>;
   showMain: () => void;
   forceRender: boolean;
   parameters: Parameters;
+  targetDOMNode: HTMLElement;
+  id: string;
 }) {
-  showMain();
```

Comment:
This is what is tripping up Chromatic -- we need to call `showMain()` at some point to remove the `.sb-show-nopreview` class from the body.

Reply:
Nice catch!

Reply:
Nice, I must have lost it I don't know how